pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "csm-test-utils"
-version = "0.2.2"
+version = "0.3.0"
```

Comment:
Is this really major version change? If there are some dependent packages with `^0.2.X` restriction, it will break them

Reply:
if it really strict binded, i'd change it